doc/user.rst

```diff
@@ -163,13 +142,14 @@ This is it. We can now use them from Matrix Depot::
     5.55112e-17  -2.77556e-17   1.94289e-16   1.0           1.38778e-16
     -6.93889e-17  -5.55112e-17  -1.66533e-16   1.38778e-16   1.0 
 
-We can also add group information in generator.jl::
+We can also add group information with::
 
-    include_generator(Group, :random, randsym)
-    include_generator(Group, :symmetric, randsym)
-    include_generator(Group, :random, randorth)
+    MatrixDepot.include_generator(MatrixDepot.Group, :random, randsym)
+    MatrixDepot.include_generator(MatrixDepot.Group, :symmetric, randsym)
+    MatrixDepot.include_generator(MatrixDepot.Group, :random, randorth)
+    MatrixDepot.init()
```

Comment:
recommending to call `MatrixDepot.init()` is not a good idea, because this call is rather expensive and does much more than required. I suggest to define a new function which handles the essentials:
```
function publish_user_generators()
    insertlocal(MATRIX_DB, GeneratedMatrixData{:U}, USERMATRIXDICT)
end

Reply:
That's a great idea. Resolved in https://github.com/JuliaMatrices/MatrixDepot.jl/pull/79/commits/c64e5d435b7e1502db515c9cb20a27c007aba1d5